spec/jobs/handover_follow_up_job_spec.rb

```diff
@@ -0,0 +1,89 @@
+require "rails_helper"
+
+describe HandoverFollowUpJob, vcr: { cassette_name: "prison_api/handover_follow_up_email" } do
+  before { allow(CommunityMailer).to receive(:with).and_return(double("CommunityMailer", urgent_pipeline_to_community: nil)) }
+
+  around do |example|
```

Comment:
Might be worth double checking we need these here. We may have something in the spec helper